spec/dummy/.controlplane/release-invalid.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cpl run:detached -a $APP_NAME --image latest -- bundle exec rake db:unexistent
```

Comment:
nonexistent

Reply:
Done in e0e3fb62ec632ea71aa631f3fcb01f2a57ffcc41.